javalib/src/main/scala/java/util/regex/Matcher.scala

```diff
@@ -265,7 +265,18 @@ final class Matcher private[regex] (var _pattern: Pattern,
     this
   }
 
-  def region(start: Int, end: Int): Matcher = ???
+  // ATTENTION: This is a sub-minimal stub implementation in order
+  //            to get ScalaNative unit and scripted tests to link.
+  //            A full implementation would have required re-working
+  //            gen Match() to have an end parameter and its callers,
+  //            in particular find(start) to specify proper start and end.
+  //
+  //            This code may be replaced soon by re2s so it does not
+  //            make sense to make major changes now.
+
+  def region(start: Int, end: Int): Matcher = {
+    this
+  }
```

Comment:
Why bother with this change at at all? The implementation does nothing and won't survive for long. It's better to just use `nativeLinkStubs := true` in unit tests projects and not worry about it for the time being.

Reply:
OK, I'll close this PR and focus on adding nativeLinkStubs.  

Thank you for the direction.

Reply:
👍